tests/test_annotations.py

```diff
@@ -7,14 +7,14 @@
 import os.path
 import shutil
 import pathlib
-
-import pandas
 import pytest
-
+import numpy as np
+import logging
 import stdpopsim
 from stdpopsim import utils
 import tests
-
+import unittest
```

Comment:
I've recently [removed `unittest` imports so that we use pytest more extensively](https://github.com/popsim-consortium/stdpopsim/pull/959/commits/80470ca2008b6caa6b32de56b7c48b9e907f8c86). Pytest uses raw asserts, test classes don't need to inherit from `unittest.TestCase` anymore, and warnings are filtered using a class/method decorator. This leads to clearer tests, usually.